.travis.yml

```diff
@@ -3,17 +3,24 @@ cache:
   directories:
     - Carthage/Builds
 
+git:
+  depth: 1
+
 install:
-    # Installing all gems from bundler
-    - bundle install
-matrix: 
+  - bundle install
+
+matrix:
   include:
     - os: osx
       language: objective-c
       osx_image: xcode10
-      script: 
+      script:
         - bundle exec fastlane test
         - bundle exec danger || true
```

Comment:
Maybe add `fast-build` param in this PR?
By the way, what if instead of `fast-build` parameter in `travis.yml`, we will add a check that will let us understand that the test command was successful and we do have test coverage file?

Reply:
In separate PR - for sure.